lib/components/Layout/PageHeader.tsx

```diff
@@ -42,16 +47,50 @@ export const PageHeader = (props) => {
 
 const Settings = () => {
   const { t } = useTranslation()
+  const prizePool = usePrizePoolBySelectedNetwork()
 
   return (
     <SettingsContainer t={t} className='ml-1 my-auto' title='Settings' sizeClassName='w-6 h-6'>
       <LanguagePicker />
       <ThemeSettingsItem t={t} />
+      <BridgeTokensModalTrigger prizePool={prizePool} />
       <TestnetSettingsItem t={t} />
     </SettingsContainer>
   )
 }
 
+const BUTTON_MIN_WIDTH = 100
```

Comment:
This is quite hidden. A better solution would be merging some of the cases (bridge & swap OR bridge & swap & buy) into a single option/modal. From there we could guide users properly. We could drop the iFrame for swapping as well and just link out for the swaps. Something like the following would be a step in the right direction:

- On deposit view there's "Get Tokens" & "Help"
- "Get Tokens" opens a modal with 3 clear sections: Bridge, Swap, Buy
- Bridge & Swap lead to links. Keeping networks in mind so the user ends up on a bridge that supports avalanche for example.
- Buy ultimately leads to the transak view, but has clear explanations on what tokens the user needs to buy.